btrack/napari/sync.py

```diff
@@ -40,8 +40,8 @@ def update_config_from_widgets(
     hypothesis_model = config.hypothesis_model
     hypothesis_model.hypotheses = [
         hypothesis
-        for hypothesis in btrack.optimise.hypothesis.H_TYPES
-        if btrack_widget[hypothesis].isChecked()
+        for i, hypothesis in enumerate(btrack.optimise.hypothesis.H_TYPES)
+        if btrack_widget["hypotheses"].item(i).isSelected()
```

Comment:
`isSelected` is different to `isChecked` and `checkState` - if you have the `Qt.ItemIsSelectable` flag set, then you can click on a row in the list in the list to selected it (and the row will be highlighted and `.isSelected` will be True). So at the minute `isSelected` is `False` for all the items, and so they're all unchecked when you reload a config. Instead you can do:
```python
if bool(btrack_widget["hypotheses"].item(i).checkState())
```

Reply:
Have changed to 
```
if btrack_widget["hypotheses"].item(i).checkState() == QtCore.Qt.CheckState.Checked
```